docs/firebase_layers.js

```diff
@@ -123,21 +124,36 @@ function getLinearGradient(colorFunction) {
   }
   let gradientString = 'linear-gradient(to right';
   switch (colorFunction.currentStyle) {
-    case 0:
-      gradientString += ', white, ' + colorFunction['color'];
+    case ColorStyle.CONTINUOUS:
+      gradientString +=
+          ', rgb(255,255,255), ' + colorToRgbString(colorFunction['color']);
       break;
-    case 1:
+    case ColorStyle.DISCRETE:
       const colors = [...(new Set(Object.values(colorFunction['colors'])))];
       const percent = 100 / colors.length;
       for (let i = 1; i <= colors.length; i++) {
-        gradientString += ', ' + colors[i - 1] + ' ' + (i * percent - percent) +
-            '%, ' + colors[i - 1] + ' ' + i * percent + '%';
+        const rgb = colorToRgbString(colors[i - 1]);
+        gradientString += ', ' + rgb + ' ' + (i * percent - percent) + '%, ' +
+            rgb + ' ' + i * percent + '%';
       }
       break;
-    case 2:
-      gradientString +=
-          ', ' + colorFunction['color'] + ', ' + colorFunction['color'];
+    case ColorStyle.SINGLE:
+      const rgb = colorToRgbString(colorFunction['color']);
+      gradientString += ', ' + rgb + ', ' + rgb;
       break;
   }
   return gradientString + ')';
 }
+
+/**
+ * Converts a color to a css-recognizable rgb string.
+ * @param {string} color
+ * @return {string}
+ */
+function colorToRgbString(color) {
+  const rgbArray = colorMap.get(color);
+  if (!rgbArray) {
```

Comment:
this is an escape hatch because a lot of consumers can handle a color string?

Reply:
Could you instead/in addition add documentation to the `@param` tag for color that says what it's expected to be?

Reply:
Done